<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBAccountSettings</class>
 <widget class="QWidget" name="KBAccountSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>499</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="payeeTab">
      <attribute name="title">
       <string>Payee</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="m_usePayeeAsIsButton">
            <property name="text">
             <string>Use payee field as provided by institution</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_extractPayeeButton">
            <property name="text">
             <string>Extract payee from memo field using regular expressions</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="m_memoFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>Payee</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>Memo</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="m_memoRegExpEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="m_payeeRegExpEdit"/>
          </item>
          <item row="4" column="1">
           <widget class="KEditListWidget" name="m_payeeExceptions">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="buttons">
             <set>KEditListWidget::All</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="textLabel3">
            <property name="text">
             <string>Regular Expression for</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="downloadTab">
      <attribute name="title">
       <string>Download</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="m_transactionDownload">
         <property name="whatsThis">
          <string>The plugin usually downloads the account balance and transactions. In some circumstances, the download of transaction data is not supported for certain accounts. The following option allows to turn off the request for transaction data.</string>
         </property>
         <property name="text">
          <string>Download transactions from institution</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="1">
          <widget class="KComboBox" name="m_preferredStatementDate">
           <item>
            <property name="text">
             <string>Ask user</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last download</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>First possible</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>Default start date for statement download</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>160</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="memoTab">
      <attribute name="title">
       <string>Memo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="m_removeLineBreaksFromMemo">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove the line breaks in the memo field that have been inserted between multiple purpose fields of the transaction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Some institutions take multiple purpose fields of a transaction and treat it as a single one. In this case the line break characters inserted after each purpose field are distracting and are removed when this option is enabled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Remove line breaks from memo field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_includePayeeDetails">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add additional payee details found in the transaction to the memo.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Include the payee details in memo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>384</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_payeeRegExpEdit</tabstop>
  <tabstop>m_memoRegExpEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_extractPayeeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_memoFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
